design-components/packages/railz-components/src/components/railz-button/railz-button.tsx

```diff
@@ -63,11 +63,17 @@ export class RailzButton {
     }
   }
 
+  private openNewTab(event: any, url: string): void {
+    event.preventDefault();
+    const newTab = window.open(url, '_blank', 'noopener,noreferrer');
```

Comment:
I'm guessing that this breaks the ability to set the target. Instead of '_blank' here, it should be this.target